test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNet/BulkOperation/BulkInsertController.cs

```diff
@@ -0,0 +1,300 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Web.Http.Controllers;
+using Microsoft.AspNet.OData;
+using Microsoft.AspNet.OData.Extensions;
+using Microsoft.AspNet.OData.Routing;
+using Microsoft.OData.Edm;
+using Microsoft.Test.E2E.AspNet.OData.Common.Controllers;
+using Xunit;
+
+namespace Microsoft.Test.E2E.AspNet.OData.BulkInsert1
+{
+    public class EmployeesController : TestODataController
+    {
+        public EmployeesController()
+        {
+            if (null == Employees)
+            {
+                InitEmployees();
+            }
+        }
+
+        /// <summary>
+        /// static so that the data is shared among requests.
+        /// </summary>
+        private static IList<Employee> Employees = null;
```

Comment:
Do you really want the data shared among requests?  Won't that hurt the ability to concurrently run tests?

Especially since you reset it with each test, wouldn't it be easier to simply make the Employees a local variable in each controller method and initialize it each use?